docs/source/model_parameters.md

```diff
@@ -0,0 +1,294 @@
+
+### Example
+
+The following example is for a single parameter description.
+
+```yaml
+%YAML 1.2
+---
+title: Schema for num_gains model parameter
+version: 0.1.0
+meta_schema: simpipe-schema
+meta_schema_url: https://raw.githubusercontent.com/gammasim/simtools/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml
+meta_schema_version: 0.1.0
+name: num_gains
+description: Number of different gains the input signal gets digitized.
+data:
+  - type: int
+    unit: dimensionless
+    default: 1
+    allowed_range:
+      min: 1
+      max: 2
+instrument:
+  class: Camera
+  type:
+    - LSTN
+    - LSTS
+    - MSTN
+    - MSTS
+    - SSTS
+    - SCTS
+activity:
+  setting:
+    - SetParameterFromExternal
+  validation:
+    - ValidateParameterByExpert
+source:
+  - Initial instrument setup
+simulation_software:
+  - name: sim_telarray
+```
+
+### Valid Keys
+
+Valid keys are described in detail in [model_parameter_and_data_schema.metaschema.yml](https://github.com/gammasim/simtools/blob/main/simtools/schemas/model_parameter_and_data_schema.metaschema.yml). The most important parameters are:
+
+#### Header section
+
+- `title`: Title of the schema file.
+- `version`: Version of this schema file.
+- `meta_schema`: Name of the base schema.
+- `meta_schema_url`: URL of the base schema.
+- `meta_schema_version`: Version of the base schema.
+
+#### Parameter description
+
+- `name`: Name of the parameter.
+- `description`: Description of the parameter.
+- `short_description`: Short description of the parameter (optional).
+
+#### Parameter data
+
+The `data` field is used to describe the actual type, format, and allowed values of the parameter.
+
+- `type`: Data type of the parameter.
+- `unit`: Units of the parameter (compatible with astropy units). Note that units are explicitly listed in the jsonschema.yml file.
+- `default`: Default value of the parameter.
+- `allowed_range`: Allowed range of the parameter.
+- `allowed_values`: Allowed values of the parameter. Use if values are not continuous and restricted to a small list of allowed values.
+
+Tables of any dimensions can be described using the `data_table` field. Example:
+
+```yaml
+data:
+  - type: data_table
+    table_columns:
+      - name: wavelength
+        description: Wavelength.
+        required: true
+        unit: nm
+        type: double
+        required_range:
+          min: 300.0
+          max: 700.0
+        input_processing:
+          - remove_duplicates
+          - sort
+      - name: photo_detection_efficiency
+        description: Average quantum or photon detection efficiency.
+        required: true
+        unit: dimensionless
+        type: double
+        allowed_range:
+          min: 0.0
+          max: 1.0
+```
+
+Input processing in form of sorting, removing of duplicates, etc. can be specified using the `input_processing` field.
```

Comment:
I am not entirely sure what that does, probably due to my lack of expertise. Once we input data, the schema file changes the data? or maybe some step is missing in the explanation?

Reply:
Sorting and duplicate removal is done (and warnings are printed). So yes, the data is changed.